fragments/HeaderFragments.ts

```diff
@@ -10,6 +10,7 @@ export class HeaderFragment {
     }
 
     async proceedToCheckout(): Promise<void> {
+        await this.cartIcon.waitFor({ state: 'visible'});
```

Comment:
It is redundant. Please check doc https://playwright.dev/docs/actionability#introduction. Playwright waits be default